GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -2102,8 +2096,8 @@ public static void main(String[] args) {
         GraphicalSurveySummaryExporter exporter = new GraphicalSurveySummaryExporter();
         Map<String, String> criteria = new HashMap<String, String>();
         Map<String, String> options = new HashMap<String, String>();
-//        options.put(TYPE_OPT, DATA_CLEANING_TYPE);
-        options.put(TYPE_OPT, DATA_ANALYSIS_TYPE);
+        options.put(TYPE_OPT, DATA_CLEANING_TYPE);
+//        options.put(TYPE_OPT, DATA_ANALYSIS_TYPE);
```

Comment:
Is the commented out code necessarry?

Reply:
Yes. That code is only used when debugging locally, and then you uncomment one of the three report types.